Wikipedia/Code/TabsOverviewCoordinator.swift

```diff
@@ -34,6 +34,56 @@ final class TabsOverviewCoordinator: Coordinator {
         self.dataController = dataController
     }
     
+    private func surveyViewController() -> UIViewController {
+        
+        var wikimediaProject: WikimediaProject? = nil
+        if let siteURL = dataStore.languageLinkController.appLanguage?.siteURL,
+        let project = WikimediaProject(siteURL: siteURL) {
+            wikimediaProject = project
+        }
+        
+        let subtitle = WMFLocalizedString("tabs-survey-title", value: "Help improve the tabs feature. Are you satisfied with this feature?", comment: "Title for article tabs survey")
+        
+        let surveyLocalizedStrings = WMFSurveyViewModel.LocalizedStrings(
+            title: CommonStrings.activityTabSurveyTitle,
+            cancel: CommonStrings.cancelActionTitle,
+            submit: CommonStrings.surveySubmitActionTitle,
+            subtitle: subtitle,
+            instructions: nil,
+            otherPlaceholder: CommonStrings.activityTabSurveyAdditionalThoughts
+        )
+
+        let surveyOptions = [
+            WMFSurveyViewModel.OptionViewModel(text: CommonStrings.activityTabSurveyVerySatisfied, apiIdentifer: "1"),
+            WMFSurveyViewModel.OptionViewModel(text: CommonStrings.activityTabSurveySatisfied, apiIdentifer: "2"),
+            WMFSurveyViewModel.OptionViewModel(text: CommonStrings.activityTabSurveyNeutral, apiIdentifer: "3"),
+            WMFSurveyViewModel.OptionViewModel(text: CommonStrings.activityTabSurveyUnsatisfied, apiIdentifer: "4"),
+            WMFSurveyViewModel.OptionViewModel(text: CommonStrings.activityTabSurveyVeryUnsatisfied, apiIdentifer: "5")
+        ]
+
+        let surveyView = WMFSurveyView(viewModel: WMFSurveyViewModel(localizedStrings: surveyLocalizedStrings, options: surveyOptions, selectionType: .single, shouldShowMultilineText: true), cancelAction: { [weak self] in
+            
+            if let wikimediaProject {
```

Comment:
There's some activity tab metrics here, let's remove for now. We may add something similar back in (probably to a different funnel class) when we pick up instrumentation.